lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -154,7 +156,8 @@ export class SilentFlowClient extends BaseClient {
         );
         if (cacheOutcome !== CacheOutcome.NOT_APPLICABLE) {
             this.logger.info(
-                `Token refresh is required due to cache outcome: '${cacheOutcome}'`
+                `Token refresh is required due to cache outcome: '${cacheOutcome}'`,
+                correlationId || ""
```

Comment:
correlationId is required here
```suggestion
                correlationId
```